ios/FluentUI/SegmentedControl/SegmentPillButton.swift

```diff
@@ -35,9 +35,13 @@ class SegmentPillButton: UIButton {
         self.contentEdgeInsets = Constants.insets
 
         let title = item.title
-        self.setTitle(title, for: .normal)
         self.accessibilityLabel = title
         self.largeContentTitle = title
+        if let image = item.image {
+            self.setImage(image.withRenderingMode(item.isTemplateImage ? .alwaysTemplate : .automatic), for: .normal)
```

Comment:
we can't trust the clients would do the right thing?

Reply:
Do you mean for the rendering mode? We don't even do the right thing, our placeholder_20 isn't a template image. We do this in a the popup menu, also.

Reply:
Looks like we do have that image as a template, but if we don't change the rendering mode we just draw the circle in blue.

Reply:
Never mind, that must've been before I was setting the tint color, or something. I removed it and it works fine now.